src/Bicep.Core/Analyzers/Linter/Rules/MaxNumberOutputsRule.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Semantics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bicep.Core.Analyzers.Linter.Rules
+{
+    public sealed class MaxNumberOutputsRule : LinterRuleBase
+    {
+        public new const string Code = "max-outputs";
+        public const int MaxNumber = 64;
+
+        public MaxNumberOutputsRule() : base(
+            code: Code,
+            description: CoreResources.MaxNumberOutputsRuleDescription,
+            docUri: new Uri($"https://aka.ms/bicep/linter/{Code}"),
```

Comment:
>{Code} [](http://example.com/codeflow?start=58&length=6)

Would you be able to write up a small doc for each new rule? The docs should go into the `/docs/linter-rules/` directory. You can use the deleted files in [https://github.com/Azure/bicep/pull/5590](https://github.com/Azure/bicep/pull/5590) as an example of the format. I think we can skip the example part on these rules because they will be super verbose, though.

Once the PR is merged, I can create the aka.ms links for the docs.

Reply:
Sure will do. I'll add the same content as I did for the azure docs PR: https://github.com/MicrosoftDocs/azure-docs/pull/86489/files

Reply:
Wait! I didn't realize you had a docs PR out. Docs is all we need 🙂.